src/VAC/VectorAnimationComplex/VAC.cpp

```diff
@@ -540,7 +540,7 @@ void VAC::draw(Time time, ViewSettings & viewSettings)
     ViewSettings::DisplayMode displayMode = viewSettings.displayMode();
 
     // Illustration mode
-    if( (displayMode == ViewSettings::ILLUSTRATION))
+    if( displayMode == ViewSettings::ILLUSTRATION)
```

Comment:
nit: we don't leave space after "(" right?

Reply:
correct, but we also don't run clang-format on vpaint because it is external code. I'll take a look at that,.